docs/contributor/benchmarks/istio-access-logs-otlp-provider.md

```diff
@@ -395,10 +422,10 @@ Observed behavior:
 memorylimiter@v0.123.0/memorylimiter.go:212    Memory usage is above soft limit. Refusing data.
 ```
 - Backpressure was not backpropagated, with the istio-proxy traffic being virtually unaffected.
+- One of the places where a difference was still perceived, was in the `istio_request_messages_total / istio_response_messages_total` metric, where a value of `29.5 / 5.15` was reported.
 - Resource consumption did not change compared to the normal working scenario.
 
-
 ## Conclusion
-Comparing the old provider (`envoy`) with the new provider (`kyma-logs`), no significant differences were observed in terms of resource consumption and performance. The new provider (`kyma-logs`) seems to be able to handle the same amount of traffic as the old provider (`envoy`), with similar CPU and memory usage and a slight increase in network bandwidth usage.
+Comparing the old provider (`envoy` and `telemetry-stdout`) with the new provider (`kyma-logs`), no significant differences were observed in terms of resource consumption and performance. The new provider (`kyma-logs`) seems to be able to handle the same amount of traffic as the old provider (`envoy` and `telemetry-stdout`), with similar CPU and memory usage and a slight increase in network bandwidth usage.
```

Comment:
It is much better no with the comparing table and using the istio metrics. However, the [Istio] Requests Total metric indicates a performance degradation which we cannot neglect and need to discuss. You are building the pure average over the 10 minutes.. can you check if that metric is really that nicely averageable, are there no bigger spikes? Also, when you repeat the test several times, how big is the variation of that value, is there a significant spreading factor?
If there is really such impact, can you maybe drop all log attributes of the provider config to see if the degradation is caused by the CEL expressions or by the OTLP communication?